reveal-ck.gemspec

```diff
@@ -15,22 +15,23 @@ Gem::Specification.new do |s|
     'A cli for generating reveal.js presentations from markdown.'
   #
   # Runtime Dependencies
-  s.add_dependency 'docile', '1.1.5'
-  s.add_dependency 'gemoji', '3.0.0'
-  s.add_dependency 'gli', '2.14.0'
+  s.add_dependency 'docile', '1.3.2'
```

Comment:
Locking these gems to a specific bugfix version in the `gemspec` might not be what we want to do. At the very least, maybe we lock them to the minor version, with flexibility for bugfix? That way, users like me who check in their `Gemfile.lock` could upgrade to a bugfix version to resolve security vulnerabilities. That would look something like `~> x.x` for each of these.

Better yet, I wonder if we allow anything under the next major version? e.g.  `['>= 2.2.0', '< 3.0']`. That would allow users to opt into newer library features, only requiring PRs to this project for major releases.

What do you think?

Reply:
Sure, we can try that. I'll go for the most liberal.

I'll consider making a note on how to use `Gemfile` / `Gemfile.lock` as you describe to avoid what happened the last time I widened the library dependencies-- bugs emerged in libraries reveal-ck depended on over time and people would have to check-in a `Gemfile.lock` to pin a library at a prior release.